geomstats/geometry/hyperbolic.py

```diff
@@ -488,7 +489,8 @@ def __init__(self, dimension, point_type='extrinsic', scale=1):
         self.scale = scale
 
     def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
-        """Compute the inner product of two tangent vectors at a base point.
+        """
```

Comment:
Please add a short description (see https://github.com/geomstats/geomstats/blob/master/docs/contributing.rst#guidelines-for-writing-documentation).